packages/graphql/src/translate/utils.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { joinStrings, isString, arrayfy, filterTruthy } from "../utils/utils";
+import { CypherStatement, CypherParams } from "./types";
+
+/** Wraps a string in a CALL statement */
+export function wrapInCall(statement: string, withVarName: string, returnStatement = "RETURN COUNT(*)"): string {
+    const withStatement = `WITH ${withVarName}`;
+    return joinStrings([withStatement, "CALL {", withStatement, statement, returnStatement, "}"]);
```

Comment:
I'm happy to add this, although keep in mind that this will not make the output completely prettified as `statement` may be a multi-line statement, only the first line will be indented.

IMO We should consider separating the query composition and its visual aspect at some point (e.g prettify at the end)